packages/farcaster-js-neynar/docs/classes/v1.CastApi.md

```diff
@@ -101,7 +101,7 @@ BaseAPI.configuration
 
 ### allCastsInThread
 
-▸ **allCastsInThread**(`requestParameters`, `options?`): `Promise`<`AxiosResponse`<[`AllCastsInThreadResponse`](../interfaces/v1.AllCastsInThreadResponse.md), `any`\>\>
+▸ **allCastsInThread**(`requestParameters`, `options?`): `Promise`<`AxiosResponse`<`AllCastsInThreadResponse`, `any`\>\>
```

Comment:
Why did all these links in the docs files break?

Reply:
still trying to figure this out

Reply:
the same thing happened in my other PR too

Reply:
When I updated the precommit to run 

```
yarn workspaces foreach --all run generate:openapi-axios
yarn workspaces foreach --all run generate:openapi-typescript
yarn workspaces foreach --all run fixup-imports
yarn workspaces foreach --all run generate:docs
``` 
instead of just
```
yarn workspaces foreach --all run generate
```

that ran `yarn workspaces foreach --all run generate:docs` for the first time in a while for the neyanr workspace. The generate command doesn't invoke the md-magic command currently - see [package.json](https://github.com/standard-crypto/farcaster-js/blob/develop/packages/farcaster-js-neynar/package.json#L86)

```
"generate": "yarn run generate:openapi",
"generate:docs": "md-magic --path './README.md' && typedoc --plugin typedoc-plugin-markdown",
"generate:openapi": "yarn run generate:openapi-axios && yarn run generate:openapi-typescript && yarn run fixup-imports",
```

That said, it still shouldn't just be removing the relative links. The hub-rest workspace still has relative links, so still investigating.